src/components/changePasswortPage/changePasswortPage.comp.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
```

Comment:
In terms of structure, can you move all auth related components to a sub folder called `auth`?